lms/djangoapps/mobile_api/users/views.py

```diff
@@ -151,7 +161,7 @@ def _get_course_info(self, request, course):
         Returns the course status
         """
         path = self._last_visited_module_path(request, course)
-        path_ids = [str(module.location) for module in path]
+        path_ids = [str(module.location) for module in path if module is not None]
```

Comment:
I suspect what you really want is to add `if course_module is None: return []` in `_last_visited_module_path` right? We shouldn't be returning `[None]` from that method.